c2corg_ui/templates/route/view.html

```diff
@@ -202,11 +203,11 @@ <h1 class="routes">
     ${show_delete_document_dialog('routes', lang, other_langs)}
   </div>
 
-    ${photoswipe_gallery()}
+  ${photoswipe_gallery()}
 
   <script>
     window.onload = function() {
-      ## init float-buttons tooltips
+      // init float-buttons tooltips
```

Comment:
``##`` are the comment sign in mako templates => this line will not appear in the generated route page.
If you use ``//`` (JS comment sign), it will appear but I am not sure it is better!?

Reply:
You're right. I didn't pay attention... I'll correct it back!